docs/source/athena-user-guide.rst

```diff
@@ -11,7 +11,7 @@ Concepts
 Querying Data
 -------------
 
-After completion of the `Athena Setup <athena-setup.html>`_ and `Athena Deploy <athena-deploy.html>`_, alerts generated from StreamAlert are now searchable in the Athena UI.
+Adding an output for rules that sends alerts generated by StreamAlert to the ``alerts`` S3 bucket will allow them to be searchable in the Athena UI.
```

Comment:
Isn't this output going to be automatically added to all rules?

Reply:
Yes good call out - but that change will happen in the next round of changes (pt 2 of this) so I just updated to doc to reflect reality for now.